pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/vedlegg/opplysningerbruktiberegningufoere/OpplysningerOmBarnetillegg.kt

```diff
@@ -0,0 +1,986 @@
+                                "blir omregnet til et årlig beløp som tilsvarer",
+                                "er",
+                            ),
+                    Nynorsk to "50 prosent av inntekta som overstig fribeløpet for ".expr()
+                            + ifElse(harTilleggForFlereFellesbarn, "barna", "barnet") +
+                            " som bur med begge foreldra sine ".expr() +
+                            ifElse(
+                                inntektEllerFribeloepErPeriodisert,
+                                "blir rekna om til et årleg beløp som svarer til",
+                                "er",
+                            ),
+                    English to "50 percent of income that exceeds the exemption amount ".expr() +
+                            ifElse(
+                                inntektEllerFribeloepErPeriodisert,
+                                "is recalculated to an annual amount of",
+                                "is",
+                            ),
+                )
+
+                showIf(avkortningsbeloepAarFelles.greaterThan(0)) {
+                    textExpr(
+                        Bokmal to " ".expr() + avkortningsbeloepAarFelles.format() + " kroner.",
+                        Nynorsk to " ".expr() + avkortningsbeloepAarFelles.format() + " kroner.",
+                        English to " NOK ".expr() + avkortningsbeloepAarFelles.format() + ".",
+                    )
+                }.orShow {
+                    textExpr(
+                        Bokmal to " ".expr() + avkortningsbeloepAarSaerkull.format() + " kroner.",
+                        Nynorsk to " ".expr() + avkortningsbeloepAarSaerkull.format() + " kroner.",
+                        English to " NOK ".expr() + avkortningsbeloepAarSaerkull.format() + ".",
+                    )
+                }
+                showIf(not(harJusteringsbeloep)) {
+                    text(
+                        Bokmal to " Dette beløpet bruker vi til å redusere dette barnetillegget for hele året.",
+                        Nynorsk to " Dette beløpet bruker vi til å redusere dette barnetillegget for heile året.",
+                        English to " This amount will be used to reduce your child supplement during the calendar year."
+                    )
+                }
+            }
+        }
+    }
+
+    data class PeriodisertInntektOverFribeloepEttTillegg(
+        val avkortningsbeloepAarFelles: Expression<Kroner>,
+        val avkortningsbeloepAarSaerkull: Expression<Kroner>,
+        val harJusteringsbeloep: Expression<Boolean>,
+        val inntektEllerFribeloepErPeriodisert: Expression<Boolean>,
+    ) : OutlinePhrase<LangBokmalNynorskEnglish>() {
+        override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+            paragraph {
+                textExpr(
+                    Bokmal to "50 prosent av den inntekten som overstiger fribeløpet ".expr() +
+                            ifElse(
```

Comment:
Det er noe med disse inline ifElse som ikke leser veldig bra, bortsett fra i de helt enkleste tilfellene som flertall/entall.

Reply:
Ja. Det leser bedre. Når jeg leser så er det må jeg finne riktig sted i 1 av 6 ting om jeg følger teksten for ett språk. Det er lettere å først se på en av to grupper med tekst, så er posisjonen på bokmåk nynorsk og engelsk veldig intuitiv. Endrer.